pynest/nest/tests/arbor_backend_nest_side.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# arbor_backend_nest_side.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+# This script runs NEST and transmits spikes via the Arbor backend.
+# Spikes are also stored to file for testing.
+# It requires NESTIO + ARBOR-NESTIO.
```

Comment:
Change file name to `arbor_backend_test_nest_side.py` and add a comment here that this file is run by `test_arbor_backend.py` together with `arbor_backend_test_arbor_side.py`.